Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/functioncompare/MultiFunctionComparisonPanel.java

```diff
@@ -317,7 +317,7 @@ public Component getListCellRendererComponent(JList<?> list, Object value, int i
 				// It's possible during a close program operation to have this 
 				// renderer called with a null value. If so, we can't get the 
 				// function so just use the default renderer.
-				return super.getListCellRendererComponent(list, value, index, isSelected,
+				return super.getListCellRendererComponent(list, null, index, isSelected,
```

Comment:
Is this change necessary?

Reply:
See https://github.com/NationalSecurityAgency/ghidra/pull/1755#discussion_r407316432